src/System.CommandLine/Help/HelpBuilder.Default.cs

```diff
@@ -75,28 +75,44 @@ public static string GetArgumentUsageLabel(Symbol parameter)
                 _ => throw new InvalidOperationException()
             };
 
-            static string? GetUsageLabel(string? helpName, Type valueType, List<Func<CompletionContext, IEnumerable<CompletionItem>>> completionSources, Symbol symbol, ArgumentArity arity)
+            static string? GetUsageLabel(
+                string? helpName,
+                Type valueType,
+                List<Func<CompletionContext, IEnumerable<CompletionItem>>> completionSources,
+                Symbol symbol,
+                ArgumentArity arity)
             {
-                // Argument.HelpName is always first choice
                 if (!string.IsNullOrWhiteSpace(helpName))
                 {
                     return $"<{helpName}>";
                 }
-                else if (
-                    !(valueType == typeof(bool) || valueType == typeof(bool?))
-                    && arity.MaximumNumberOfValues > 0 // allowing zero arguments means we don't need to show usage
-                    && completionSources.Count > 0)
-                {
-                    IEnumerable<string> completions = symbol
-                        .GetCompletions(CompletionContext.Empty)
-                        .Select(item => item.Label);
 
-                    string joined = string.Join("|", completions);
+                if (valueType == typeof(bool) ||
+                    valueType == typeof(bool?) ||
+                    arity.MaximumNumberOfValues <= 0) // allowing zero arguments means we don't need to show usage
+                {
+                    return null;
+                }
 
-                    if (!string.IsNullOrEmpty(joined))
+                if (completionSources.Count <= 0)
+                {
+                    if (symbol is Option)
                     {
-                        return $"<{joined}>";
+                        return $"<{symbol.Name.Trim('-', '/')}>";
```

Comment:
nit: should we use remove both the leading and trailing chars?

```suggestion
                        return $"<{symbol.Name.TrimStart('-', '/')}>";
```